src/common/tiles/MovieTile/Genre/genreSaga.js

```diff
@@ -0,0 +1,25 @@
+import { call, delay, put, takeLatest } from "redux-saga/effects";
```

Comment:
Mistrzem API nie jestem, możecie mnie wyśmiać potem xD, ale kurczę czy naprawdę do tych generes trzeba było tworzyć całe osobne Sagi i Slice? Nie można było tego dodać  w np. popularMovieSaga?

Reply:
Akurat do genres to wydaje mi sie, ze spoko zrobic osobna sage i slice bo Genre to jest osobny komponent i do Genre bedziemy przekazywac tylko konretne id w referencji, wiec chyba w ten sposob jest to bardziej zgodne ze sztuka i uporzadkowane

Reply:
Hmmm no ok ;)